### PR TITLE
🎨 Mobile version

### PR DESCRIPTION
**Pull request by mamadOuologuem**
- Mobile version update 

Screen Shot 2019-09-01 at 3 42 49 PM

### CODE REVIEW

File: src/lib/callApi.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ try {
 } catch (e) { console.error(e); }
 
 // If eq 'dev' fetch data locally else (eq 'prod') fetch data from Firebase
-const SOURCE = 'prod';
+const SOURCE = 'dev';
```
Comment (ericboucher) [inline src/lib/callApi.js]: was this change intended?

Comment (mamadOuologuem) [reply src/lib/callApi.js]: Fixed with a .env variable
